src/MensaWidget/MWParserTest.class.st

```diff
@@ -0,0 +1,105 @@
+"
+This class is the test class for the MWParser.
+"
+Class {
+	#name : #MWParserTest,
+	#superclass : #TestCase,
+	#category : #'MensaWidget-Tests',
+	#'squeak_changestamp' : 'Max 5/29/2024 08:53'
+}
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Max 5/28/2024 18:21'
+}
+MWParserTest class >> multipleJSONs [
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Max 5/28/2024 18:42'
+}
+MWParserTest class >> simpleJSON [
+	^ '[{"id":19500904,"name":"testName"}]'
+]
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Max 5/28/2024 18:16'
+}
+MWParserTest >> setUp [
+	
+]
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Max 5/28/2024 18:16'
+}
+MWParserTest >> tearDown [
+]
+
+{
```

Comment:
i believe if you dont use those functions you can just delete them all together

Reply:
yeah, true. Will do that.
I can implement tests for the MWMeal, but i think it is important to have tests for a generic object, not just our use case.